### PR TITLE
Removed hover, deleted grayscale. Added proper indent to the code and added <script>files to the <head> tag.

### PR DESCRIPTION
**Pull request by siddhant2001**
The code is cleaner, there is no hover effect, page looks nice and simple.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
-    <link rel="stylesheet" type="text/css" href="stylesheets/bootstrap.min.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/main.css" media="screen">
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script src="js/bootstrap.min.js"></script>
```
Comment (hemantjadon) [inline index.html:5]: The first line in the head should be the `meta tags` not the `render-blocking` js and css loading

File: stylesheets/stylesheet.css [hunk 2]
```diff
@@ -878,4 +882,3 @@ footer a:hover {
 
 /* Mobile Portrait Size to Mobile Landscape Size (devices and browsers) */
 @media only screen and (max-width: 479px) { }
```
Comment (hemantjadon) [inline stylesheets/stylesheet.css:884]: Add the proper EOL/EOF character `[CRLF]`. (Simply a new line at the end).

File: index.html [hunk 3]
```diff
@@ -1,19 +1,28 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
+  <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
+  <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
+  <link rel="stylesheet" type="text/css" href="stylesheets/bootstrap.min.css" media="screen">
+  <link rel="stylesheet" type="text/css" href="stylesheets/main.css" media="screen">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (hemantjadon) [inline index.html:18]: Why double loading of `bootstrap.min.js` one at top in head and one here

**Review by hemantjadon — CHANGES_REQUESTED** [review]
Also Please provide the Task link and the link to your gh-pages of your fork.

**Comment by hemantjadon** [comment]
Also please check for merge conflicts

File: stylesheets/stylesheet.css (hunk 2 above)
Comment (siddhant2001) [reply stylesheets/stylesheet.css:884]: That is not my code.

Comment (hemantjadon) [reply stylesheets/stylesheet.css:884]: The line was there before your commit.
And every line of code is ours not of any particular person.
If there is a bug and you can fixit you should.

File: index.html (hunk 1 above)
Comment (siddhant2001) [reply index.html:5]: The code doesn't work when I do that. Please let it be, the website works fine.

File: index.html (hunk 3 above)
Comment (siddhant2001) [reply index.html:18]: By mistake, removed it.

File: stylesheets/stylesheet.css (hunk 2 above)
Comment (siddhant2001) [reply stylesheets/stylesheet.css:884]: I don't know what EOL/EOF character [CRLF] is.

Comment (hemantjadon) [reply stylesheets/stylesheet.css:884]: Simply add a newline at the end of the file and it will be done.

Comment (siddhant2001) [reply stylesheets/stylesheet.css:884]: Ok

File: CNAME [hunk 4]
```diff
@@ -1 +1 @@
-gci16.fossasia.org
```
Comment (hemantjadon) [inline CNAME:1]: Why changing the removing the correct CNAME?

**Review by hemantjadon — CHANGES_REQUESTED** [review]

**Comment by hemantjadon** [comment]
Please check the merge conflicts also

**Review by jigyasa-grover — COMMENTED** [review]
@siddhant2001 

* Squash all your commits into one and rename it to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.
* Resolve merge conflicts, this [How To Manually Fix Git Merge Conflicts (video)](https://www.youtube.com/watch?v=g8BRcB9NLp4) might be helpful. 

Cheers !

File: CNAME (hunk 4 above)
Comment (jigyasa-grover) [reply CNAME:1]: I'd suggest you to read more about `CNAME`.
You arent supposed to change it :)

Links:
* https://help.github.com/articles/using-a-custom-domain-with-github-pages/
* https://en.wikipedia.org/wiki/CNAME_record

**Review by hemantjadon — APPROVED** [review]

**Comment by jigyasa-grover** [comment]
Wrt [rules in CONTRIBUTING.md](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md), closing this.

**Pull request closed without merging**